kolibri/core/assets/src/core-getters.js

```diff
@@ -32,6 +32,10 @@ function getDefaultChannelId(channelList) {
 
 /* return the current channel object, according to vuex state */
 function getCurrentChannelObject(state) {
```

Comment:
This fixes an issue on the management page (content) that would show the error box `Channel not found` since there were no available channels.